FullyNoded/Helpers/FirstTime.swift

```diff
@@ -8,6 +8,10 @@
 
 class FirstTime {
     
+    enum Event: String {
```

Comment:
FirstTime has the explicit purpose of creating an encryption key and that is it, there should not be anything to do with utxos here.